docs/user-guide/simple-yaml.md

```diff
@@ -31,7 +31,7 @@ can be code reviewed, producing a more robust, reliable and archival system.
 
 ```bash
 cd kubernetes
-kubectl create -f pod.yaml
+kubectl create -f /path/to/pod.yaml
```

Comment:
I started with /path/to/ and realized for most cases ./ was easier.  will fix this.